display/d.histogram/pie.c

```diff
@@ -29,9 +29,10 @@
  */
 
 #include <string.h>
-
+#include <grass/gis.h>
 #include <grass/raster.h>
 #include <grass/display.h>
+#include <grass/glocale.h>
```

Comment:
Sorry for nitpicking, please order these alphabetically.

Reply:
I know order of imports matter in C, since when not well isolated, different order gives different results. For my knowledge, are the headers in grass side-effect free with respect to different import order?

Reply:
They should be, or else we'll find out... :-)

Reply:
Done